src/drivers/imu/invensense/icm20948/icm20948_i2c_passthrough_main.cpp

```diff
@@ -51,7 +51,7 @@ I2CSPIDriverBase *ICM20948_I2C_Passthrough::instantiate(const BusCLIArguments &c
 		int runtime_instance)
 {
 	ICM20948_I2C_Passthrough *instance = new ICM20948_I2C_Passthrough(iterator.configuredBusOption(), iterator.bus(),
-			cli.bus_frequency);
+			cli.bus_frequency, cli.i2c_address);
```

Comment:
`cli.rotation` needs to be passed as well if you want to make use of that.

Reply:
It seems that IMU unit reading is not implemented in the pass-trought driver. And this parameter (rotation) is no longer used anywhere. 

So for now, I would suggest merge it like this. And leave the compass reading via I2C (in passtrought mode) alone - and in a separate PR. I can delete this adjustment from it, if it's a problem.

Reply:
Yes please remove it then, and also `PRINT_MODULE_USAGE_PARAM_INT('R', 0, 0, 35, "Rotation", true);`

Reply:
Ok @bkueng, I have removed it in #d4a1802